func_sig_registry/registry/views.py

```diff
@@ -92,13 +132,10 @@ def post(self, *args, **kwargs):
         serializer = self.get_serializer(data=self.request.data)
         if not serializer.is_valid():
             return Response({'serializer': serializer})
-        signature = serializer.save()
+        results = serializer.save()
```

Comment:
I think you should update the serializer so that it returns 2 tuples of the results:

```python
function_signatures, event_signatures = serializer.save()
message = ...  # construct the message based on the returned signatures
messages.success(...)
```

Having the message construction embedded in the serializer isn't intrinsicly wrong, but it does mix concerns where they probably shouldn't be mixed.  The serializer can be purely in charge of validating and saving the data, and then you can use the returned data to construct the message.